dotCMS/src/main/java/com/dotcms/publisher/pusher/PushUtils.java

```diff
@@ -76,8 +76,10 @@ private static void addFilesToCompression(TarArchiveOutputStream taos, File file
 				if (file.isFile()) {
 			        // Add the file to the archive
 					BufferedInputStream bis = new BufferedInputStream(new FileInputStream(file));
-					IOUtils.copy(new FileInputStream(file), taos);
+					FileInputStream in = new FileInputStream(file);
```

Comment:
is it the same FileInputStream used in line 78? If so, the object can be reused. 
I suggest to handle input stream this way: https://dzone.com/articles/fileinputstream-fileoutputstream-considered-harmful